Submitty/Submitty
# [Feature:InstructorUI] Teaching Staff Locked Gradeables Access

### Please check if the PR fulfills these requirements:


### What is the current behavior?

Currently locked gradeables were previously inaccessible for teaching staff and instructors. 
This would prevent students from getting help with certain gradeables if teaching staff hasn't completed the prerequisite.
When teaching staff attempts to access locked graded the below alert pops up.
<img width="449" alt="Screenshot 2023-02-28 at 2 25 36 PM" src="https://user-images.githubusercontent.com/122416028/221958005-f87b2c9a-ba39-45d0-9416-4d153225035c.png">
Closes #8355 
### What is the new behavior?

Now locked gradeables can be bypassed with a confirmation screen allowing teaching staff to access the submission page.
After clicking ok teaching staff will be redirected to the submission page.
<img width="451" alt="Screenshot 2023-02-28 at 3 23 19 PM" src="https://user-images.githubusercontent.com/122416028/221969862-cb89c33d-2baa-4ce8-8c65-2d08c05a34ff.png">

### Other information?
To Test:
1.) Create a gradeable that requires another gradable as a prerequisite.
2.) Attempt to access the as a student and verify that there is no access to that gradeable.
3.) Verify all teaching staff (Mentors/TAs/Instructors) can access the gradeable after click ok on the conformation prompt.
